### ISSUE TITLE
Disable buttons in two-page view

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fleszar1/flexpaper#354]
```
1. Open Adaptive UI interface, flash version
2. Switch to Two page view

Fit size buttons and zoom slider behaviours are buggy. They should be disabled. 


FlexPaper version: FlexPaper_2.0.6_c_Annotations_1.0.20_Customizable_UI
OS: Ubuntu 12.04
Browser: Chromium 24
```

Original issue reported on code.google.com by `federico...@gmail.com` on 5 Feb 2013 at 11:00

**Comment by GoogleCodeExporter** [comment]
```
Fixed in upcoming build 2.1.0
```

Original comment by `erik.eng...@devaldi.com` on 12 Feb 2013 at 11:28
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
The error persists when switching to two-page mode, hence buttons are not 
disabled. 

http://www.hyperborea.com/www-share/Dante/ld/two-page-error.ogv

FlexPaper version: FlexPaper_2.1.0_c_Annotations_1.0.40_Customizable_UI-rc2
OS: Ubuntu 12.04
Browser: Chromium 24
```

Original comment by `federico...@gmail.com` on 20 Feb 2013 at 8:33
